src/ape/api/networks.py

```diff
@@ -526,20 +525,8 @@ def decode_error(self, abi: ErrorABI, data: HexBytes, **kwargs) -> ContractLogic
         """
 
         # TODO: Include ErrorABI in official `decode_calldata` API definition for 0.7
```

Comment:
Is this still accurate?

Reply:
`decode_calldata` should also permit `ErrorABI` as a valid input type, ya